BraveShared/Preferences.swift

```diff
@@ -226,6 +226,13 @@ extension Preferences {
     public func reset() {
       value = defaultValue
     }
+    
+    public func notify() {
+      let key = self.key
+      observers.forEach {
+        $0.preferencesDidChange(for: key)
+      }
+    }
```

Comment:
No but I have a rare case where I want to force a notification otherwise I have to write more notification code :(
Basically, I have an observer that when the preference changes it updates all the webViews with the new zoom level. Instead of duplicating this for when a `specific` web-page's zoom level changes, I just trigger the same observer. I'm really not sure if i should just write separate code for it or reuse what's already there and just call `notify` which I added.

Reply:
This seems weird though, and maybe needs a different solution. You're notifying the observers that something has changed when nothing has, so I'd say this function should definitely not exist

Reply:
Fixed. Used NotificationCenter.